README.md

```diff
@@ -3,88 +3,85 @@
-* Do not be afraid to introduce large changes, but only if they respect previous interfaces
+* When building interfaces accessible by other apps/users, try and stick to a pre-defined set of standards and schemas (e.g. swagger). Make the schemas discoverable, ideally referenced in the README.md on oda-hub repositories.
+* Follow the [issue handling principles](https://github.com/oda-hub/doc-ops-reporting#issue-handling-principles) - in order to never have user issues unhandled.
+* Do not be afraid to introduce large changes, but only if they respect previous interfaces.
 
 ## Do not forget to
 
-we all do many things, but we should be concious that some actions are needed for smooth progress of collective project.
-Even though these actions may be delayed as necessary, it should be done with intent.
-Too much notifications and distractions are detrimental to productivity, but one should aim keep some level of engagement.
+We all do many things, but we should be concious that certain actions are crucial for the progress of collective projects: even though those may be delayed if necessary, there should always be a reason behind. Too much notifications and distractions are detrimental to productivity, but some level of engagement should  always be maintaned.
 
-It may be advisable to make sure all actions and channels converge to email. 
+It may be advisable to make sure all actions and channels converge to email and notifications, in order to keep everybody aware of the ongoing changes:
 
 * on github check assigned issues, reviews, pull requests (this is all reflected in notifications, and other fields in the github page header)
 * communicate by mattermost and slack
 
 ## Branches, Versions, Releases
 
-### What did we use before and what's wrong with it
+### Our previous approach and the relateive problems
 
 We used release branches for each version, like **production-V.V**. Any bugfixes were applied to these branches.
-In addition, **staging-V.V** were used for ongoing developments in current version. Also, for previous versions, **staging-V.V** were used to preview updates before making them live.
-This worked rather well.
+In addition, **staging-V.V** were used for ongoing developments in current version. Also, for previous versions, **staging-V.V** were used to preview updates before making them live. This worked quite well.
 
-On github, it is not convenient that current development is in the latest **staging-V.V** branch, whichever it happens to be at the moment. To reduce the uncertainty on what is the current version in development, we can just use the **master** branch for merging all new features.
+This approach is not convenient on github, so to reduce the uncertainty on what is the current development version, we can just use the **master** (or **main**) branch, and merge on it all the new features.
```

Comment:
I have specified also **main** since I see sometimes one in use instead of the other, and I don't know which one should be used. I thought this was a change that was decided from github, and a complete transition to main from master was happening. I am absolutely fine with using any of the two, we just have to agree.

Reply:
github changed to main, and gitlab will also transition [eventually](https://about.gitlab.com/blog/2021/03/10/new-git-default-branch-name/). Exactly, we need to keep one and than change. This is exactly the place to say which. We used master so far. So let's write "master" until agreed to change all at once? Saying  "master or main" is problematic since it is not clear which.

Reply:
Yes makes sense. I put it there since github is confusing me a bit on which one will be used by default (for example I see that in this repo there are both). but let's stick with "master" for now, and we'll change when needed.

Reply:
indeed, and we can also mention, separately that we might want to move to main later.